src/Microsoft.ML.Data/DataView/TypedCursor.cs

```diff
@@ -217,6 +217,97 @@ public RowCursor<TRow>[] GetCursorSet(Func<int, bool> additionalColumnsPredicate
                  .ToArray();
         }
 
+        public static bool ValidateMemberInfo(MemberInfo memberInfo, IDataView data)
+        {
+            if (!SchemaDefinition.CheckMemberInfo(memberInfo))
+                return false;
+
```

Comment:
`ValidateMemberInfo`  throws in some cases and returns false in one case. Can we make that consistent? Should it either always throw or always return false upon failure?

Reply:
have made it consistent